types/read_config.go

```diff
@@ -92,9 +108,15 @@ func (ReadConfig) Read(hasEnv HasEnv) BootstrapConfig {
 
 // BootstrapConfig for the process.
 type BootstrapConfig struct {
-	EnableFunctionReadinessProbe bool
-	ReadTimeout                  time.Duration
-	WriteTimeout                 time.Duration
-	ImagePullPolicy              string
-	Port                         int
+	HTTPProbe                         bool
+	ReadinessProbeInitialDelaySeconds int
+	ReadinessProbeTimeoutSeconds      int
+	ReadinessProbePeriodSeconds       int
```

Comment:
Curious if this is worth making a Golang duration in this PR or a follow-up?

The other configs support Golang durations now, could call `durationVal.Seconds()` in the code to convert if that makes sense.

Reply:
I don't consider this as compulsory - just want your take on it.